docker/configure_workers_and_start.py

```diff
@@ -770,7 +794,15 @@ def generate_worker_files(
 
     # A list of internal endpoints to healthcheck, starting with the main process
     # which exists even if no workers do.
-    healthcheck_urls = ["http://localhost:8080/health"]
+    # This list ends up being part of the command line to curl, which added unix socket
+    # support in version 7.40. The scheme and hostname are ignored, the path is not.
+    if using_unix_sockets:
+        healthcheck_urls = [
+            f"--unix-socket {MAIN_PROCESS_UNIX_SOCKET_PUBLIC_PATH} "
+            "http://localhost/health"
```

Comment:
We could move this part of the comment directly above `http://localhost/health` to better associate it with what it's talking about:

```suggestion
            # The scheme and hostname from the following URL are ignored.
            # The only thing that matters is the path `/health`
            "http://localhost/health"
```

Reply:
Oh yes, I like that. I forget you can do that with Python.